src/main/resources/config/liquibase/changelog/20230207120000_changelog.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.18.xsd">
+    <changeSet author="b-fein" id="20230207120000">
+        <modifyDataType tableName="feedback" columnName="detail_text" newDataType="text"/>
```

Comment:
On a side note, for the communication feature (posts) the content column was recently changed to `varchar(5000)`. Should we keep it consistent?
https://github.com/ls1intum/Artemis/blob/4ce6ff19328ed67f4320b7e298d19dd70768ae7e/src/main/resources/config/liquibase/changelog/20230201102700_changelog.xml